### ISSUE TITLE
AltspaceVR: World Editor page for SDK apps should hide URI when using appID

### ISSUE THREAD

**Issue by sorenhan** [microsoft/mixed-reality-extension-sdk#655]
The AltspaceVR world editor is currently allowing users to change the URI if an MRE has been created using a registered app. This shouldn't be allowed - if it is a registered app, the appID should be used to look up the URI from ALtspace's app register every time

There are 2 separate changes needed 
1. In the World editor page to change settings on an MRE, if the MRE has an App ID then the world editor should disable the URI editor
2. During MRE instantiation, if an MRE has an App ID, then the App ID should be used to look up the URI, rather than trust the URI coming in from the space component

**Comment by stevenvergenz** [comment]
One question mark with this issue (which I agree with in general) is how query arguments should be handled. Some (many?) apps rely on them to be useful.

**Comment by SorenHannibal** [comment]
good point. Maybe instead of hiding the field it could show mre:12345678?parameters or  ws://uri?parameters, depending on if it is using an appid or a uri.

**Comment by stevenvergenz** [comment]
I don't think we should expose the app ID to the users. Maybe we can make the URL a label instead of a text field, but keep a text field for the query params?

**Comment by sorenhan** [comment]
Good idea to separate the two. There already is a separate label field for all MREs, so maybe we could separate parameters and URL, and just hide URL field when we have an appID.

**Comment by tombuMS** [comment]
Agree with Steven on the approach in separating the two.
